dbms/src/Storages/KVStore/Utils/AsyncTasks.h

```diff
@@ -53,7 +53,24 @@ struct AsyncTasks
         , log(DB::Logger::get())
     {}
 
-    ~AsyncTasks() { LOG_INFO(log, "Pending {} tasks when destructing", count()); }
+    void shutdown()
+    {
+        LOG_INFO(log, "AsyncTasks: Pending {} tasks when destructing", count());
```

Comment:
nit: the logger will log down the filename, so the "AsyncTasks: " here is just duplicated

Reply:
fixed